less/variables.less

```diff
@@ -18,7 +18,7 @@
 @redSidebarColor: #BB3C33;
 @pinkSidebarColor: #D64262;
 @footerBackgroundColor: #f1f1f1;
-@mozFestPromoRed: #E12526;
+@thimblePromoGreen: #279761;
```

Comment:
yeah that seems a better idea. That we way we won't have to rename it in the future